hastexo/provider.py

```diff
@@ -370,6 +404,11 @@ def delete_stack(self, name, wait=True):
         try:
             self.logger.info("Deleting OpenStack Heat stack [%s]" % name)
             self.heat_c.stacks.delete(stack_id=name)
+            self.nova_c.keypairs.delete(name)
+        except NotFound:
+            # Nova not finding a keypair to delete is a valid scenario,
+            # if the key is handled by heat via the lab template.
+            pass
```

Comment:
If we replace this `pass` with a call to `logger.info()`, then we get some traces in the logs if the key pair can't be found. This will come in handy when we eventually move from Heat-generated to XBlock-generated private keys, when such a log message reminds us of courses where the lab hasn't yet been updated.

Reply:
Sure, makes sense, done.